src/composables/useNotifications.ts

```diff
@@ -1,11 +1,22 @@
-import { useStore } from 'vuex';
+import { ref } from 'vue';
+
+interface Notification {
+  message: string;
+  type: string;
+  timestamp: number;
+}
 
 export function useNotifications() {
-  const store = useStore();
+  const items = ref<Notification[]>([]);
```

Comment:
I think `items` needs to be outside of the function for the state to be persistent.